### PR TITLE
Mark IDeltaManager.close() as deprecated

### PR DESCRIPTION
**Pull request by markfields**
Jump-starting this for the sake of #6936  (too bad we didn't do this months ago!)

### CODE REVIEW

File: common/lib/container-definitions/src/deltas.ts [hunk 1]
```diff
@@ -165,7 +165,7 @@ export interface IDeltaManager<T, U> extends IEventProvider<IDeltaManagerEvents>
 
     readonly readOnlyInfo: ReadOnlyInfo;
 
-    /** Terminate the connection to storage */
```
Comment (vladsud) [inline common/lib/container-definitions/src/deltas.ts:168]: It's not deprecated in https://github.com/microsoft/FluidFramework/blob/release/0.45/common/lib/container-definitions/src/deltas.ts, and I believe 0.45 is our LTS branch now.
It's not deprecated in 0.47, and 0.47 is one of the most used loader version, if I remember.

It maybe all fine as what matters - there are no runtime versions in prod that are using it - can you please make sure it's actually the case?

**Review by vladsud — APPROVED** [review]

File: common/lib/container-definitions/src/deltas.ts (hunk 1 above)
Comment (markfields) [reply common/lib/container-definitions/src/deltas.ts:168]: I don't understand your concern.  I'm just marking it as deprecated now, we will remove it in a few months I figure.

Comment (vladsud) [reply common/lib/container-definitions/src/deltas.ts:168]: You are right, never mind :)

Comment (markfields) [reply common/lib/container-definitions/src/deltas.ts:168]: But for the record, I don't see any usages in main right now, so that's a good sign.

**Pull request merged**